processing/src/test/java/org/apache/druid/segment/UnnestColumnValueSelectorCursorTest.java

```diff
@@ -54,7 +51,7 @@ public void test_list_unnest_cursors()
         listCursor.getColumnSelectorFactory(),
         "dummy",
         OUTPUT_NAME,
-        IGNORE_SET
+        null
```

Comment:
Do these tests a wide range of predicates (filters) on the unnested value? I worry that the code above handles only simple cases: `unnested_value op constant`. Do they also handle complex cases such as `fn(unnested_value) > some_other_col`? `fn(unnested_value) > 10 AND unnested_value * 3 > 12`? Would be good to have tests that exercise these non-trivial cases.